ide/nbproject/project.xml

```diff
@@ -112,6 +112,18 @@
                     <location>${project.dir}/test/simulator/test</location>
```

Comment:
In the `cassandra.classpath.jars` element above, do we need to continue to include the `harry-core` JAR?

Reply:
We do not! Removed.